config/initializers/simple_form.rb

```diff
@@ -66,7 +72,7 @@
   config.error_notification_tag = :div
 
   # CSS class to add for error notification helper.
-  config.error_notification_class = 'alert alert-error'
+  config.error_notification_class = 'error_notification'
```

Comment:
@Gataniel it depends on styles that we use in forms. As I know we use bootstrap and classes `alert alert-error` will show red colored alerts. Please check it and use styles that will works from Twitter Bootstrap.